google/cloud/security/common/data_access/sql_queries/select_data.py

```diff
@@ -149,11 +149,24 @@
     WHERE project_number = {1};
 """
 
-
-SELECT_VIOLATIONS = """
+SELECT_POLICY_VIOLATIONS = """
     SELECT * FROM violations_{0};
 """
 
+SELECT_BIGQUERY_ACL_VIOLATIONS = """
+    SELECT * FROM violations_{0}
+    WHERE violation_type = 'BIGQUERY_VIOLATION';
```

Comment:
I think there is a way to reduce these sql to a generic statement, where the violation type can be specified as {1}.  It might take some work, so please put a TODO.